ts/defs.bzl

```diff
@@ -252,10 +252,10 @@ def ts_project(
     """
 
     if srcs == None:
-        include = ["**/*.ts", "**/*.tsx"]
+        include = ["**/*.ts", "**/*.mts", "**/*.cts", "**/*.tsx"]
```

Comment:
I don't think this is required for the fix? This seems more like a feature that would need testing...